pyiceberg/io/__init__.py

```diff
@@ -62,13 +70,13 @@
 HDFS_PORT = "hdfs.port"
 HDFS_USER = "hdfs.user"
 HDFS_KERB_TICKET = "hdfs.kerberos_ticket"
-ADLFS_CONNECTION_STRING = "adlfs.connection-string"
-ADLFS_ACCOUNT_NAME = "adlfs.account-name"
-ADLFS_ACCOUNT_KEY = "adlfs.account-key"
-ADLFS_SAS_TOKEN = "adlfs.sas-token"
-ADLFS_TENANT_ID = "adlfs.tenant-id"
-ADLFS_CLIENT_ID = "adlfs.client-id"
-ADLFS_ClIENT_SECRET = "adlfs.client-secret"
+ADLFS_CONNECTION_STRING = "adls.connection-string"
+ADLFS_ACCOUNT_NAME = "adls.account-name"
+ADLFS_ACCOUNT_KEY = "adls.account-key"
+ADLFS_SAS_TOKEN = "adls.sas-token"
+ADLFS_TENANT_ID = "adls.tenant-id"
+ADLFS_CLIENT_ID = "adls.client-id"
+ADLFS_ClIENT_SECRET = "adls.client-secret"
```

Comment:
How about 
```python
ADLS_CONNECTION_STRING = "adls.connection-string"
...
```
for new `adls.*` properties and leave `ADLFS_*` properties unchanged to remain backward compatible? This could make the constants naming more consistent.